Sources/App/Views/Sections/Editor/EditorSection.swift

```diff
@@ -14,15 +14,18 @@ struct EditorSection: View {
   /// The app environment object.
   @EnvironmentObject var appEnvironment: AppEnvironment
 
+  /// The editor environment object.
+  @EnvironmentObject var editorEnvironment: EditorEnvironment
```

Comment:
If we will change the environment to EditorSectionEnvironment we should change this property too

```suggestion
  /// The editor section environment object.
  @EnvironmentObject var editorSectionEnvironment: EditorSectionEnvironment
```